pkg/minikube/node/start.go

```diff
@@ -297,6 +297,9 @@ func setupKubeAdm(mAPI libmachine.API, cfg config.ClusterConfig, n config.Node,
 	// update cluster and set up certs
 
 	if err := bs.UpdateCluster(cfg); err != nil {
+		if errors.Is(err, cruntime.ErrContainerRuntimeNotRunning) {
+			exit.Message(reason.EnvDockerUnavailable, "Container runtime is not running.")
```

Comment:
Please create a new Reason
EnvDockerUnavilable is NOT same as RunTime not running

try this, create a new Reason
``
if err := bs.UpdateCluster(cfg); err != nil {
		if errors.Is(err, cruntime.ErrContainerRuntimeNotRunning) {
                          exit.Error(reason.KubernetesInstallFailedRuntimeNotRunning, "Failed to update cluster", err)
           }
```